include/wx/qt/textctrl.h

```diff
@@ -8,10 +8,33 @@
 #ifndef _WX_QT_TEXTCTRL_H_
 #define _WX_QT_TEXTCTRL_H_
 
-class QLineEdit;
-class QTextEdit;
 class QScrollArea;
 
+class wxQtEdit
```

Comment:
This is an implementation helper, so it really should be defined in `qt/textctrl.cpp` and not here, wx users don't need to see it at all.

Also, it would be nice to have a comment explaining that this class virtualizes the behaviour of all these methods between single- and multi-line text controls.

Reply:
Done. I had wanted to move this to a private header, but was concerned that this would violate style or structure further. I'm happy to put it in the cpp file.